ng-pkgs/activation_service/package.toml

```diff
@@ -1,3 +1,19 @@
 name = "activation_service"
 is_auth = false
-is_admin = false
\ No newline at end of file
+is_admin = false
+
+
+[[servers]]
+name = "activation_root_proxy"
+domain = "gettft.testnet.grid.tf"
+letsencryptemail = "tarekr@incubaid.com"
+includes = ["default_443.chatflows_*", "default_443.auth_*", "default_443.activation*"]
+
+[[servers.locations]]
+name = "custom"
+type = "custom"
+custom_config = """
+location /activation_service {
+    rewrite /activation_service/(.*)$ /activation_service/actors/activation_service/$1;
```

Comment:
The path in the jsx-implementation is /threefoldfoundation/activation_service/

Reply:
in jsx actors were accessible using `author/package_name/actors/actor_name/method`, but in js-ng the package is idnetified using its name only and so the actor of the package is accessible through `package_name/actors/actor_name/method`

Reply:
I understand but we since it's a rewrite we can keep the incoming url the same

Reply:
okay

Reply:
Modified